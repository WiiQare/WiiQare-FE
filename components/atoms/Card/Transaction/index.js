import { CiCircleList, CiSaveDown2 } from "react-icons/ci";

import DropdownFilter from "../../../atoms/Dropdown/Filter";
import { Button } from "flowbite-react";
const CardTransaction = () => {
  return (
    <div className="border w-full rounded-lg py-8 px-6 flex flex-col flex-wrap md:flex-row justify-between gap-4 bg-white cursor-pointer">
      <div className="">
        <h1 className="md:text-2xl text-gray-700 font-bold flex-wrap flex items-center gap-2 text-lg">
          Transactions History
        </h1>
      </div>
      <div className="flex flex-row flex-wrap  gap-2 center justify-end space-x-2">
        <DropdownFilter
          label="Sort By"
          className="w-[auto]"
          labelClassName="py-1 w-[auto] border-0"
          icon={() => <span className="h-[15px] w-[15px] rounded bg-primary" />}
          items={["Detail", "Cancel"]}
        />
        <Button className="bg-primary">
          <CiSaveDown2 className="mr-2 h-5 w-5" />
          Download Report
        </Button>
        <DropdownFilter
          label="Sort By"
          className="w-[auto]"
          labelClassName="py-1 w-[auto]"
          icon={() => <CiCircleList />}
          items={["Detail", "Cancel"]}
        />
      </div>
    </div>
  );
};

export default CardTransaction;
