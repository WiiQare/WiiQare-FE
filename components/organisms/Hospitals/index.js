import React, { createContext, useState } from 'react';
import CardHeader from '../../atoms/Card/Header';
import { CiCircleList } from 'react-icons/ci';
import DataTable from 'react-data-table-component';
import { useSession } from 'next-auth/react';
import Fetcher from '../../../lib/Fetcher';
import { FcAddressBook, FcCellPhone } from 'react-icons/fc';
import Link from 'next/link';

const businessTypes = {
  MEDICAL_CABINET: { text: 'Cabinet Médical', color: '447698' },
  CLINIC: { text: 'Clinique', color: 'A143D3' },
  HOSPITAL: { text: 'Hôpital', color: 'FF5498' },
};

const columns = [
  {
    name: "IDENTITÉ DE L'HÔPITAL",
    selector: (row) => (
      <div className="flex py-4 gap-4 w-fit">
        <div className="min-w-20 w-20 h-fit border p-3 rounded-lg flex justify-center items-center">
          <img src={row.logoLink} alt="" />
        </div>

        <div className="flex flex-col gap-1">
          <span className="uppercase font-semibold text-md">{row.name}</span>

          <span className="text-xs text-gray-500">{row.email}</span>

          <span className="text-xs text-gray-500">{row.address}</span>

          <span className="flex gap-3 items-center text-sm text-gray-700 font-light mt-3">
            <span className="bg-orange opacity-90 px-2 py-1 text-white text-xs rounded-md w-fit flex gap-3 items-center">
              {row.packages ? row.packages.length : 0} Plan
              {row.packages.length > 1 ? 's' : ''}
            </span>
          </span>
        </div>
      </div>
    ),
  },

  {
    name: 'DÉTAILS BUSINESS',
    selector: (row) => (
      <div className="flex flex-col gap-4">
        <span
          className={`bg-gray-300 text-gray-900 py-1 pl-2 font-semibold w-fit pr-4 rounded-full opacity-80 `}
        >
          {businessTypes[row.businessType].text}
        </span>
        <div className="flex flex-col gap-1">
          <Link href={`tel:${row.phone}`} legacyBehavior>
            <span className="text-xs text-gray-500 flex items-center gap-1 cursor-pointer">
              <FcCellPhone size={18} /> {row.phone}
            </span>
          </Link>

          <Link href={`mailto:${row.email}`} legacyBehavior>
            <span className="text-xs text-gray-500 flex items-center gap-1 cursor-pointer">
              <FcAddressBook size={18} /> {row.email}
            </span>
          </Link>
        </div>
      </div>
    ),
  },

  {
    name: 'ACTIONS',
    button: true,
    cell: (row) => (
      <div>
        <button
          className="border border-gray-800 hover:bg-gray-800 hover:text-gray-200 font-semibold px-3 py-2 text-gray-800 text-xs rounded-md"
          onClick={(e) => alert(row.id)}
        >
          Packages
        </button>
      </div>
    ),
  },
];

const Hospital = () => {
  let { data: session } = useSession();
  const { data, isLoading, isError } = Fetcher(
    `/provider/list`,
    session.user.data.access_token,
  );

  return (
    <div className="p-2 space-y-6 md:py-8 md:px-6 mb-20 md:mb-6">
      <CardHeader
        title={'Liste des hôpitaux'}
        filter={{
          label: {
            title: 'Trier par',
            className: 'py-1 w-[auto]',
          },
          className: 'w-[auto]',
          icon: () => <CiCircleList />,
          items: ['Detail', 'Annuler'],
        }}
      />

      <div className="border rounded-lg overflow-x-auto w-full relative">
        {isLoading ? (
          <>Loading...</>
        ) : (
          <>
            {data.length > 0 ? (
              <DataTable
                columns={columns}
                data={data}
                fixedHeader
                className="px-8 bg-white"
                title="Liste des hôpitaux"
                persistTableHead
              />
            ) : (
              <div className="py-10 mx-auto w-1/2 flex flex-col justify-center items-center gap-4">
                <p className="text-gray-500 text-sm">
                  Aucun hôpital actuellement ajouté...
                </p>
              </div>
            )}
          </>
        )}
      </div>
    </div>
  );
};

export default Hospital;
