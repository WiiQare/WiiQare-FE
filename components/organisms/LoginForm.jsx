import React from "react";
import Link from "next/link";
import TextField from "@mui/material/TextField";
import Box from "@mui/material/Box";
import Stepper from "@mui/material/Stepper";
import Step from "@mui/material/Step";
import StepButton from "@mui/material/StepButton";
import Button from "@mui/material/Button";
import Stack from "@mui/material/Stack";
import useMediaQuery from "@mui/material/useMediaQuery";
import Visibility from "@mui/icons-material/Visibility";
import VisibilityOff from "@mui/icons-material/VisibilityOff";

import OnboardingScreen from "../molecules/OnboardingScreen";
import { useLocalStorage } from "../../hooks/useLocalStorage";
import {
  FormControl,
  IconButton,
  InputAdornment,
  InputLabel,
  OutlinedInput,
  Typography,
} from "@mui/material";
import MenuHolder from "../atoms/MenuHolder";
import BlinkSnackbar from "../atoms/BlinkSnackbar";

function LoginForm() {
  const matches = useMediaQuery("(max-width: 992px)");
  const [onboardingIsEnd, setOnboardingIsEnd] = React.useState("");
  const [showPassword, setShowPassword] = React.useState(false);

  const handleClickShowPassword = () => setShowPassword((show) => !show);

  React.useEffect(() => {
    setOnboardingIsEnd(localStorage.getItem("onboardingIsEnd") || "");
  }, []);

  const formHolder = (
    <>
      <div className="form-holder">
        <MenuHolder href="/register" label="SIGN UP" />
        <div className="signin-signup-form">
          <div className="form-items">
            <div className="form-title">Sign in to your account</div>
            <Box sx={{ mb: 3, mt: 2 }}></Box>
            <form id="signinform">
              <Stack spacing={2}>
                <TextField
                  id="outlined-basic"
                  fullWidth
                  label="E-mail Address"
                  variant="outlined"
                />
                <FormControl fullWidth variant="outlined">
                  <InputLabel htmlFor="outlined-basic1">Password</InputLabel>
                  <OutlinedInput
                    id="outlined-basic1"
                    label="Password"
                    type={showPassword ? "text" : "password"}
                    endAdornment={
                      <InputAdornment position="end">
                        <IconButton
                          aria-label="toggle password visibility"
                          onClick={handleClickShowPassword}
                          edge="end"
                        >
                          {showPassword ? <VisibilityOff /> : <Visibility />}
                        </IconButton>
                      </InputAdornment>
                    }
                  />
                </FormControl>
                <Box sx={{ textAlign: "right", mb: 2 }}>
                  <Typography color="text.secondary" variant="caption">
                    Forgot your password ?
                  </Typography>
                </Box>
                <Box>
                  <Button
                    size="large"
                    variant="contained"
                    onClick={() => {}}
                    className="bg-yellow"
                  >
                    SIGN IN
                  </Button>
                </Box>
              </Stack>
            </form>
          </div>
        </div>
      </div>
      <BlinkSnackbar />
    </>
  );
  return (
    <>
      {matches && !onboardingIsEnd ? (
        <OnboardingScreen
          onStartClick={() => {
            if (typeof window !== "undefined") {
              localStorage.setItem("onboardingIsEnd", "onboardingIsEnd");
            }
            setOnboardingIsEnd("onboardingIsEnd");
          }}
        />
      ) : (
        formHolder
      )}
    </>
  );
}

export default LoginForm;
