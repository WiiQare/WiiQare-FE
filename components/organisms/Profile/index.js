import Image from "next/image";
import CardHeader from "../../atoms/Card/Header";
import { BiCamera } from "react-icons/bi";
import Tabs from '@mui/material/Tabs';
import Tab from '@mui/material/Tab';
import Box from '@mui/material/Box';
import PropTypes from 'prop-types';
import Typography from '@mui/material/Typography';
import { useState } from "react";
import { FormControl, InputLabel, MenuItem, Select, TextField } from "@mui/material";

const TabHistories = [
	{
		name: "About"
	},

	{
		name: "Settings",
	}
]

const Profile = () => {
	const [value, setValue] = useState(0);

	const handleChange = (event, newValue) => {
		setValue(newValue);
	};
	return (
		<div className="p-2 space-y-6 md:py-8 md:px-6 mb-20">
			<CardHeader
				title={"My Profile"}
				breadcrumbs={[
					{
						item: "Home",
						link: "/"
					},
					{
						item: "John Doe",
						link: "/profile"
					}
				]}
				download={false}
				print={false}
			/>

			<section className="w-full flex flex-col gap-8 items-start pb-20 md:pb-0">
				<div className="w-full overflow-hidden md:col-span-2 rounded-lg p-4 flex flex-col gap-6 bg-white drop-shadow-sm">
					<div className="bg-[url(https://i.goopics.net/a5yedr.jpg)] bg-no-repeat h-56 rounded-lg"></div>

					<div className="flex justify-between items-center md:px-6">
						<div className="flex gap-5 items-center relative">
							<div className="w-20 h-20">
								<Image src={"https://xsgames.co/randomusers/avatar.php?g=male"} width={80} height={80} className="object-cover rounded-full h-full w-full" />
							</div>

							<div className="">
								<h1 className="text-sky font-bold text-xl">Mitchell C. Shay</h1>
								<span className="text-xs">frdrcpeter@gmail.com</span>
							</div>
						</div>

						<div>
							<button htmlFor="avatar" className="bg-[rgb(29,170,230,.6)] p-2 border-2 border-sky rounded-md text-white hover:bg-[rgb(29,170,230,.8)] w-fit">
								<BiCamera size={22} />
							</button>

							<input type="file" name="" id="avatar" className="hidden" />
						</div>
					</div>
				</div>

				<div className="md:w-3/4 w-full">
					<div className="min-full md:col-span-3 rounded-lg p-4 flex flex-col gap-4 bg-white drop-shadow-sm">
						<Box sx={{ width: '100%' }}>

							<Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
								<Tabs value={value} onChange={handleChange} aria-label="basic tabs example">
									{TabHistories.map((item, index) => <Tab label={item.name} {...a11yProps(index)} key={index} />)}
								</Tabs>
							</Box>

							<div className="min-w-full">
								{TabHistories.map((item, index) => <TabPanelContent value={value} index={index} />)}
							</div>
						</Box>
					</div>
				</div>
			</section>
		</div>
	);
};

export default Profile;

export function TabPanel(props) {
	const { children, value, index, ...other } = props;

	return (
		<div
			role="tabpanel"
			hidden={value !== index}
			id={`simple-tabpanel-${index}`}
			aria-labelledby={`simple-tab-${index}`}
			{...other}
		>
			{value === index && (
				<Box sx={{ p: 3 }}>
					<Typography>{children}</Typography>
				</Box>
			)}
		</div>
	);
}

TabPanel.propTypes = {
	children: PropTypes.node,
	index: PropTypes.number.isRequired,
	value: PropTypes.number.isRequired,
};

function a11yProps(index) {
	return {
		id: `simple-tab-${index}`,
		'aria-controls': `simple-tabpanel-${index}`,
	};
}

function TabPanelContent({ value, index }) {
	return (
		<TabPanel value={value} index={index} >
			{
				index == 0 ? <About /> : <Settings />
			}
		</TabPanel>
	)
}

function About() {
	return (
		<section className="space-y-8">
			<h2 className="text-sky font-semibold">Personal Information</h2>

			<div className="md:w-3/6 space-y-4">
				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Name :</h5>
					<span className="text-gray-500">Mitchell C. Shay</span>
				</div>

				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Email :</h5>
					<span className="text-gray-500">frdrcpeter@gmail.com</span>
				</div>

				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Availability :</h5>
					<span className="text-gray-500">Full time</span>
				</div>

				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Age :</h5>
					<span className="text-gray-500">37</span>
				</div>

				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Location :</h5>
					<span className="text-gray-500">Rosemont Avenue Melbourne, Florida</span>
				</div>

				<div className="grid grid-cols-2">
					<h5 className="font-semibold">Year Experience :</h5>
					<span className="text-gray-500">07 Year Experiences</span>
				</div>
			</div>
		</section>
	)
}

function Settings() {
	const [state, setState] = useState('');

	const handleState = (event) => {
		setState(event.target.value);
	};
	return (
		<section className="space-y-8">
			<h2 className="text-sky font-semibold">Account Setting</h2>

			<div className="flex md:grid md:grid-cols-2 gap-8">
				<TextField
					fullWidth
					type={"email"}
					className="placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="Email Address"
					placeholder="Email Address"
					name="email"
					variant="outlined"
				/>

				<TextField
					fullWidth
					type={"password"}
					className="placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="Password"
					name="password"
					placeholder="Password"
					variant="outlined"
				/>
			</div>

			<div className="">
				<TextField
					fullWidth
					type={"text"}
					className="placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="Adress"
					placeholder="123 Main Street"
					name="email"
					variant="outlined"
				/>
			</div>

			<div className="">
				<TextField
					fullWidth
					type={"text"}
					className="placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="Adress 2"
					placeholder="123 Main Street"
					name="email"
					variant="outlined"
				/>
			</div>

			<div className="flex md:hidden">
					<FormControl fullWidth>
						<InputLabel id="demo-simple-select-label">State</InputLabel>
						<Select
							labelId="demo-simple-select-label"
							id="demo-simple-select"
							value={state}
							label="State"
							onChange={handleState}
						>
							<MenuItem value={"Kinshasa"} selected={true}>Kinshasa</MenuItem>
							<MenuItem value={"Goma"}>Goma</MenuItem>
							<MenuItem value={"Lubumbashi"}>Lubumbashi</MenuItem>
						</Select>
					</FormControl>
				</div>


			<div className="flex md:grid md:grid-cols-4 gap-8">
				<TextField
					fullWidth
					type={"text"}
					className="col-span-3 md:col-span-2 placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="City"
					placeholder="City"
					name="city"
					variant="outlined"
				/>

				<div className="hidden md:flex">
					<FormControl fullWidth>
						<InputLabel id="demo-simple-select-label">State</InputLabel>
						<Select
							labelId="demo-simple-select-label"
							id="demo-simple-select"
							value={state}
							label="State"
							onChange={handleState}
						>
							<MenuItem value={"Kinshasa"} selected={true}>Kinshasa</MenuItem>
							<MenuItem value={"Goma"}>Goma</MenuItem>
							<MenuItem value={"Lubumbashi"}>Lubumbashi</MenuItem>
						</Select>
					</FormControl>
				</div>

				<TextField
					fullWidth
					type={"text"}
					className="placeholder:text-gray-400 hover:outline-none focus:ring-0 border border-gray-300 rounded-lg focus:ring-sky"
					label="ZIP"
					name="zip"
					placeholder="ZIP"
					variant="outlined"
				/>
			</div>
		</section>
	)
}