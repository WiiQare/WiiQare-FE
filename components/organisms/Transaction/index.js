import React, { createContext, useState } from "react";
import CardHeader from "../../atoms/Card/Header";
import HistoryWallet from "../Wallet/History";
import { CiCircleList } from "react-icons/ci";

import Quickly from "./quickly";
export const TransactionContext = createContext();


const Transaction = () => {

	const [transaction, setTransaction] = useState({state: false, transactions: []});

	return (
		<div className="p-2 space-y-6 md:py-8 md:px-6 mb-20">
			<CardHeader
				title={"Historique des transactions"}
				sort={{
					icon: () => <span className="h-[15px] w-[15px] rounded bg-primary" />,
					items: ["DÃ©tail", "Annuler"]
				}}
				filter={{
					label: {
						title: "Trier par",
						className: "py-1 w-[auto]"
					},
					className: "w-[auto]",
					icon: () => <CiCircleList />,
					items: ["Detail", "Annuler"]
				}}
			/>

			<TransactionContext.Provider value={{ transaction, setTransaction }}>
				<HistoryWallet />
				<Quickly />
			</TransactionContext.Provider>

		</div>
	);
};

export default Transaction;
