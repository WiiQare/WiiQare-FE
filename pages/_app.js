import Head from "next/head";
import { SessionProvider } from "next-auth/react";
import { ThemeProvider } from "@mui/material/styles";
import { theme } from "../theme";

import "../styles/globals.css";
import "../styles/main.css";

export default function App({ Component, pageProps }) {
  const getLayout = Component.getLayout ?? ((page) => page);
  return (
    <>
      <Head>
        <meta
          name="description"
          content="Payé les soins à vos familles depuis l'étranger"
        />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="icon" href="/favicon.ico" />
        <script src="https://unpkg.com/flowbite@1.5.3/dist/flowbite.js"></script>

      </Head>
      <SessionProvider session={pageProps.session}>
        <ThemeProvider theme={theme}>        
          {getLayout(<Component {...pageProps} />)}
        </ThemeProvider>
      </SessionProvider>
    </>
  );
}
