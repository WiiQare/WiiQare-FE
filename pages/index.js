import Head from 'next/head'
import { getSession, useSession, signOut } from "next-auth/react"

export default function Home() {

	const { data:session } = useSession()
	
	const handleSignOut = () => {
		signOut()
	}

	return (
		<>
			<Head>
				<title>JAMII - UNICEF CRYPTO</title>
				<meta name="description" content="Payé les soins à vos familles depuis l'étranger" />
				<meta name="viewport" content="width=device-width, initial-scale=1" />
				<link rel="icon" href="/favicon.ico" />
			</Head>
			

			{session ? <User session={session} handleSignOut={handleSignOut} /> : <Guest />}
		</>
	)
}

function User({session, handleSignOut}) {
	return (
		<main>
			Homepage
		</main>
	)
}

function Guest() {
	return (
		<main className='container mx-auto text-center py-20 space-y-8'>
			<h3 className="text-4xl font-bold">Guest User</h3>
		</main>
	)
}

export async function getServerSideProps({req}) {
	const session = await getSession({req})

	if(!session) {
		return {
			redirect: {
				destination: '/login',
				permanent: false
			}
		}
	}

	return {props: {session}}
}