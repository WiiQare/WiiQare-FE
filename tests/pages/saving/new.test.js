import NewSaving from '@/pages/saving/new';
import { render } from '@testing-library/react';
import { TransactionContext } from '@/components/organisms/Transaction';
import { SessionProvider } from 'next-auth/react';
import { DrawContext } from '../../../pages/_app';
import { QueryClientProvider, QueryClient } from 'react-query';

jest.mock('next/router', () => ({
  useRouter: jest.fn().mockReturnValue({
    router: [],
  }),
}));

describe('Saving', () => {
  it('should render', () => {
    const queryClient = new QueryClient();
    const { container } = render(
      <QueryClientProvider client={queryClient}>
        <DrawContext.Provider value={{ draw: {}, setDraw: () => {} }}>
          <SessionProvider session={{ user: { data: { access_token: {} } } }}>
            <TransactionContext.Provider
              value={{ transaction: {}, setTransaction: () => {} }}
            >
              <NewSaving />
            </TransactionContext.Provider>
          </SessionProvider>
        </DrawContext.Provider>
      </QueryClientProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
