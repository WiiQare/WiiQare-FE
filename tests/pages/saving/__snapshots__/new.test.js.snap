// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Saving should render 1`] = `
<div>
  <div
    class="p-2 space-y-6 md:py-8 md:px-6 mb-20 md:mb-10"
  >
    <div
      class="border w-full rounded-lg py-8 px-6 flex flex-col flex-wrap md:flex-row justify-between items-center gap-4 bg-white cursor-pointer"
    >
      <div
        class="flex flex-col"
      >
        <h1
          class="md:text-2xl text-gray-700 font-bold flex-wrap flex items-center gap-2 text-lg"
        >
          Nouvelle épargne
        </h1>
        <ul
          class="flex gap-2"
        >
          <li>
            <span
              class="flex gap-2 font-semibold text-sky"
            >
              Épargne de santé
              <span>
                /
              </span>
            </span>
          </li>
          <li>
            <span
              class="flex gap-2 font-light text-gray-500"
            >
              Séléctionner le type d'épargne
            </span>
          </li>
        </ul>
      </div>
      <div
        class="flex flex-row flex-wrap  gap-2 center justify-end space-x-2"
      />
    </div>
    <div
      class="my-10 md:my-20 grid md:grid-cols-4 grid-cols-2 gap-5 md:gap-10 items-center justify-center self-center"
    >
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/yojhn7.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Pour moi
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/cub5b2.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Famille
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/dtrv5p.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Enfant
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/gl9c62.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Femme enceinte
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/3w4x3m.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Diabétique
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="order-2 md:order-1 bg-white p-5 md:p-10 flex flex-col justify-center items-center rounded-lg shadow-sm hover:shadow-md duration-200 transition-all cursor-pointer gap-4"
      >
        <div
          class="p-3 rounded-md border border-blue-200 w-fit"
        >
          <img
            alt="Myself"
            class="w-20"
            src="https://i.goopics.net/hhbm6i.png"
          />
        </div>
        <div
          class="flex flex-col justify-center items-center"
        >
          <h3
            class="uppercase text-xs md:text-sm font-bold"
          >
            Hyper tendu
          </h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M16.72 7.72a.75.75 0 011.06 0l3.75 3.75a.75.75 0 010 1.06l-3.75 3.75a.75.75 0 11-1.06-1.06l2.47-2.47H3a.75.75 0 010-1.5h16.19l-2.47-2.47a.75.75 0 010-1.06z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="flex flex-col w-full h-full p-6 bg-white rounded-lg shadow-sm col-span-2 order-1 md:order-2"
      >
        <h5
          class="mb-2 text-xl font-bold tracking-tight text-gray-900 "
        >
          Vous éparnez pour qui, mieux pour quoi ?
        </h5>
        <p
          class="font-normal text-gray-700 text-sm"
        >
          Sélectionner un type d'épargne par ceux présenter ici, les calculs des prix d'épargne sont fait suivant le type d'épargne choisi.
        </p>
        <p
          class="font-normal text-gray-700 text-sm mt-2"
        >
          Epargner c'est vivre le futur, c'est condamner le futur à être meilleur pour vous et surtout ceux qui vous sont cher...
        </p>
      </div>
    </div>
  </div>
</div>
`;
